<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWordWindow</class>
 <widget class="QDialog" name="AddWordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodaj słowo</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>23</y>
     <width>47</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Słówko</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="eng_word_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Polskie znaczenie</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pol_word_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Przykładowe zdanie</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sentence_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Synonimy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="synonyms_edit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Anuluj</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="text">
       <string>Zapisz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
